src/org/opendatakit/briefcase/ui/MainBriefcaseWindow.java

```diff
@@ -124,63 +124,75 @@ public static void main(String[] args) {
         System.exit(1);
       }
 
-      if (cmd.hasOption(HELP)) {
-        showHelp(options);
-        System.exit(1);
-      }
+      runLegacyCli(cmd, () -> MainBriefcaseWindow.showHelp(options));
+    }
+  }
 
-      if (cmd.hasOption(VERSION)) {
-        showVersion();
-        System.exit(1);
-      }
+  public static void runLegacyCli(CommandLine cmd, Runnable helpShower) {
```

Comment:
I think referring to a launcher isn't clear. I'd prefer simpler language here.

> You are using old commands which will soon be removed from the CLI.
Please run Briefcase with --help to learn about the new commands.`